### PR TITLE
Module 6 home task has been completed.

### PR DESCRIPTION
**Pull request by Adenmin**

### CODE REVIEW

File: module5/RawParser.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        with open(self.file_path) as file:
+            lines = file.readlines()
+        return lines
+
+    def __get_pub_objects(self, lines: List[str]) -> List[Dict[str, str]]:
+        try:
+            publications = []
+            for line in lines:
+                objs = line.split(";")
+                pub = {'type': objs[0].lower().replace(" ", ""),
```
Comment (romkabauer) [inline module5/RawParser.py:28]: maybe also apply sof?

File: module5/Publisher.py [hunk 2]
```diff
@@ -60,3 +68,12 @@ def pub_builder(pub_type: PublicationType, pub_text: List[str], spec_info: str)
     def writer(pub: Publication) -> None:
```
Comment (romkabauer) [inline module5/Publisher.py]: I would rename it in more "actionable" way like write_to_newsfeed. The same for pub_builder a bit upper

File: module5/RawParser.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    def parse_raw_file(self):
+        return self.__get_pub_objects(self.__read_file())
+
+    def __read_file(self) -> List[str]:
+        with open(self.file_path) as file:
+            lines = file.readlines()
+        return lines
+
+    def __get_pub_objects(self, lines: List[str]) -> List[Dict[str, str]]:
+        try:
```
Comment (romkabauer) [inline module5/RawParser.py:24]: Maybe it worth to check more specific exceptions like Wrong number of args for publication or check for some data formats, for example if type ad -> objs[2] should have particular format, etc. But it is not necessary now, I think

File: module5/RawParser.py (hunk 1 above)
Comment (Adenmin) [reply module5/RawParser.py:28]: The sof module doesn't contain a suitable function for this transformation. The main purpose of this transformation is to cleanup any possible formatting and spaces for accurately identifying the type of publication in the file. The header text isn't taken from this file, it is formed in the code.

File: module5/Publisher.py (hunk 2 above)
Comment (Adenmin) [reply module5/Publisher.py]: Fixed.

File: module5/RawParser.py (hunk 3 above)
Comment (Adenmin) [reply module5/RawParser.py:24]: I agree that the structure of the exceptions in this code is far from perfect :)
In my opinion, building the correct structure is a little out of scope of the program and can take time comparable to an OOP module.

File: module5/RawParser.py (hunk 1 above)
Comment (romkabauer) [reply module5/RawParser.py:28]: I mean if you use functions from another module for cleaning input data (all of three properties are from file), let's implement functions for cleaning in those module and apply them in an abstract way. But it's up to you:)

**Pull request merged**